solo-project/src/actions.js

```diff
@@ -0,0 +1,18 @@
+import { apiCall } from './apiMiddleware';
+
+export const getProducts = () => ({
+  type: 'GET_PRODUCTS',
+  [apiCall]: {
+    url: 'http://private-623b1-beraiseapi.apiary-mock.com/products',

```

Comment:
Best practice: put this base url (`http://private-623b1-beraiseapi.apiary-mock.com`) in somewhere it can be reused. You'll have to change it when you move out from the mock api, it's better to do it once.